tribute_page.html

```diff
@@ -44,11 +46,5 @@ <h2>Here's a time line of Dr. Borlaug's life:</h2>
       </article>
       <hr/>
     </div>
-  </main>
-  <footer>
-    <div id="footer-content">
-      <secion>If you have time, you should read more about this incredible human being on his <a href="https://www.pbs.org/wgbh/americanexperience/features/green-revolution-norman-borlaug-race-to-fight-global-hunger/" id="tribute-link" target="_blank">American Experience</a>.
-      </section>
-    </div>
-  </footer>
```

Comment:
Great job removing unnecessary information from your source code. With time you'll develop more skills to create better and better websites.